### DISCUSSION TITLE
SQL Error [922] [42000]: ORA-00922: missing or invalid option

### DISCUSSION THREAD

**Discussion by murathptest** [dbeaver/dbeaver#13053] [Q&A]
#### System information: 
- Windows 10
- 7.2.2
- 1.8.0.271 
- Additional extensions

#### Connection specification:
- Oracle 12.2 Enterprise
- ojdbc8.jar
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 

#### Describe the problem you're observing:
When I type a single command such as
SET VERIFY OFF;
and try to execute it to a oracle environment,
I am seeing the above error

SQL Error [922] [42000]: ORA-00922: missing or invalid option

I verified that simple SELECT * is working fine...but anything with SET is creating a problem

**Comment by LonwoLonwo** [comment]
Hello @murathptest 
As I can see "SET anything" statements are SQL*Plus specific https://docs.oracle.com/cd/B19306_01/server.102/b14357/ch12040.htm

**Reply by pkkht** [reply]
Ditto same error as reported. How to run SQL script with "SET... " commands in it in DBeaver? .. I am using DBeaver CE Version 23.0.4.202305071722 on Windows 10
